sdk/containerregistry/azure-containerregistry/azure/containerregistry/_download_stream.py

```diff
@@ -11,7 +11,7 @@
 
 
 class GetNext(Protocol):
-    def __call__(self, *args: Any, range_header: str) -> Iterator[bytes]:
+    def __call__(self, *args: Any, range_header: str) -> Iterator[bytes]:  # pyright: ignore
```

Comment:
what is the pyright error?

Reply:
Oh, I forgot to remove it. Since it's hard to clean up pyright errors for this package, I'll resolve the most and will leave the rest there.

Reply:
The error is:
```console
error: Function with declared type of "Iterator[bytes]" must return value
    "__next__" is not present
    "__iter__" is not present (reportGeneralTypeIssues)
```